database/migrations/2023_06_10_193615_create_website_paypal_transactions.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('website_paypal_transactions', function (Blueprint $table) {
+            $table->id();
+
+            $table->integer('user_id');
+            $table->string('transaction_id');
+            $table->string('status')->nullable();
```

Comment:
It seems like you're trying to update the `description` column in your ShopController.php - as this is the behavior applied in https://github.com/ObjectRetros/atomcms/pull/149

However your migration is missing the description column.

```suggestion
            $table->string('status')->nullable();
            $table->string('description', 512)->nullable();
```